source/_integrations/plugwise.markdown

```diff
@@ -44,11 +44,30 @@ Platforms available - depending on your Smile and setup include:
  - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)
 
 
-The password can be found on the bottom of your Smile or Stretch, the ID, it should consist of 8 characters. To find your IP address use the Plugwise App: 
+## Before configuring
 
- - Open the Plugwise App and choose the 'Settings'-icon (&#9776;) and choose 'HTML-interface'. 
- - Go to the (lower) 'Settings'-icon (&#9776;) and choose 'Preferences'. 
- - Choose 'System' then 'Networking' and your IP address will be shown.
+{% important %}
+When you have an Anna and an Adam, only the Adam will be shown as discovered. Make sure to **only** configure the Adam integration, i.e. do **not** manually configure the Anna.
+{% endimportant %}
```

Comment:
I'm still confused about this important warning... even if they tried, it will fail; so why scare the end user here?

Reply:
Good call ... adjusted accordingly